display/display.go

```diff
@@ -60,6 +61,18 @@ var App = cview.NewApplication()
 func Init(version, commit, builtBy string) {
 	aboutInit(version, commit, builtBy)
 
+	// Detect terminal colors for syntax highlighting
+	switch termenv.ColorProfile() {
+	case termenv.TrueColor:
+		renderer.TermColor = "terminal16m"
+	case termenv.ANSI256:
+		renderer.TermColor = "terminal256"
+	case termenv.ANSI:
+		renderer.TermColor = "terminal16"
+	default:
+		renderer.TermColor = "terminal8"
```

Comment:
Code highlighting should not be happening at all in this case, as this means the terminal does not support ANSI at all.

Reply:
You're right, it's just that there's a gap since termenv can't detect 8-color terminals. On the other hand, I'm not sure how many of those exist anyway.